torch/csrc/jit/codegen/cuda/lower_unroll.cpp

```diff
@@ -54,6 +329,19 @@ bool isReductionInitExpr(const kir::Expr* expr) {
   return true;
 }
 
+bool containsMisalignedVectorization(const kir::ForLoop* fl) {
```

Comment:
From the name I would expect this to be recursive, what you're really doing is seeing if a direct child of `fl` is set as misaligned vectorize.

Reply:
I want the for-loop that contains vectorized UnaryOpType::Set operations.
Reads and writes use the same shift value.